docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1119,6 +1119,28 @@ To ensure high availability when you have multiple instances of Config Server de
 
 If you use HTTP basic security on your Config Server, it is currently possible to support per-Config Server auth credentials only if you embed the credentials in each URL you specify under the `spring.cloud.config.uri` property. If you use any other kind of security mechanism, you cannot (currently) support per-Config Server authentication and authorization.
 
+=== Config Client Properties
+
+The `spring.cloud.config.authorization` property has now been removed. It can now be provided through the additional headers used to create the client request - `spring.cloud.config.headers.authorization`. Also , two new properties have been added - `spring.cloud.config.requestReadTimeout` and `spring.cloud.config.sendState`. Please find below the complete list of client properties along with their description.
```

Comment:
I dont think we need all of this.  It is a enough to add a section on configuring read timeouts.  The properties will automatically be updated using a script when we release Finchley.

Reply:
So,If I understand correctly , no need for complete list of properties? And add a small section on configuring read timeouts? Is that right?

Reply:
yup